bullet_train-outgoing_webhooks/test/models/webhooks/outgoing/endpoint_test.rb

```diff
@@ -0,0 +1,51 @@
+require "test_helper"
+
+class Webhooks::Outgoing::EndpointTest < ActiveSupport::TestCase
+  include ActiveJob::TestHelper
+
+  setup do
+    @team = Team.create!(name: "test-team")
+    @endpoint = Webhooks::Outgoing::Endpoint.create!(url: "https://example.com/webhook", name: "test", team: @team)
+  end
+
+  test "#create should accept valid event_types" do
+    @endpoint = Webhooks::Outgoing::Endpoint.create!(
+      url: "https://example.com/webhook",
+      name: "test",
+      team: @team,
+      event_type_ids: [Webhooks::Outgoing::EventType.all.first.id]
+    )
+    assert @endpoint.persisted?
+  end
+
+  test "#create should accept non-existent event_types" do
+    @endpoint = Webhooks::Outgoing::Endpoint.create!(
+      url: "https://example.com/webhook",
+      name: "test",
+      team: @team,
+      event_type_ids: ["fake-thing.create"]
+    )
+    assert @endpoint.persisted?
+  end
```

Comment:
Yeah, maybe this was one of the reasons why the `.find()` was there, namely to raise on creation, since there is no other validation in place for that. It did make sense that we raised before here.